src/shared_modules/content_manager/src/components/updaterContext.hpp

```diff
@@ -142,6 +147,21 @@ struct UpdaterContext
      *
      */
     int currentOffset {0};
+
+    /**
+     * @brief Construct a new Updater Context object
+     *
+     */
+    UpdaterContext() = default;
+
+    /**
+     * @brief Re initialize the context with the default values.
+     *
+     */
+    void initialize()
+    {
+        *this = UpdaterContext();
+    }
```

Comment:
Note: Tests for this behaviour can be seen [here](https://godbolt.org/z/EKnTo6Mh6)